src/hyperfine/internal.rs

```diff
@@ -97,6 +103,39 @@ pub fn min(vals: &[f64]) -> f64 {
         .unwrap()
 }
 
+pub fn write_benchmark_comparison(results: &Vec<ExportEntry>) {
+    if results.len() < 2 {
+        return;
+    }
+
+    // Show which was faster, maybe expand to table later?
+    let mut fastest_item: &ExportEntry = &results[0];
+    let mut longer_items: Vec<&ExportEntry> = Vec::new();
+
+    for run in &results[1..] {
+        if let Some(Ordering::Less) = fastest_item.mean.partial_cmp(&run.mean) {
+            longer_items.push(run);
+        } else {
+            longer_items.push(fastest_item);
+            fastest_item = run;
+        }
+    }
+
+    println!("{}\n", "Result Comparison".bold());
+    println!("'{}' ran", fastest_item.command.green());
+    longer_items.sort_by(|l, r| l.mean.partial_cmp(&r.mean).unwrap_or(Ordering::Equal));
+
+    for item in longer_items {
+        println!(
+            "{} faster than '{}'",
+            format!("{:8.2}x", item.mean / fastest_item.mean)
+                .bold()
+                .blue(),
+            &item.command.red()
```

Comment:
Really minor thing: maybe use cyan/magenta instead of green/red for the commands? It would fit with the min/max coloring and I would like red to only signal errors.

Reply:
Sure, I was just picking colors that are easy for me to pick out. I've got some colorblindness and cyan/magenta/blue all look rather similar